designs/tripleo/osc-tripleo.md

```diff
@@ -119,7 +119,16 @@ outputs:
               - OSC_ENV_VAR={get_param: OSCEnvVar}
               - TripleO::Services::OSC::tls_truststore_file: {get_param:OSCTrustStoreFile}
               - TripleO::Services::OSC::keystore_file: {get_param:OSCKeyStoreFile}
-```  
+        tripleo.osc.firewall_rules:
```

Comment:
I assume these are the ports with-in the container. If so, can we hard-code the values? exposing all of these configuration to end-user might be headache....

Reply:
The ports within the container here 443 and 8090 are hardcoded in the template. The user cannot provide them as parameters when deploying the overcloud. The parameterized ports are the host ones.